Modules/Timer/TimerTests/LogSuggestions/CalendarSuggestionProviderTests.swift

```diff
@@ -0,0 +1,119 @@
+import XCTest
+import Models
+import OtherServices
+import CalendarService
+@testable import Timer
+
+class CalendarSuggestionProviderTests: XCTestCase {
+    var now = Date(timeIntervalSince1970: 987654321)
+    var mockTime: Time!
+
+    override func setUp() {
+        mockTime = Time(getNow: { return self.now })
+    }
+
+    func test_provider_returnsOnlyOneSuggestion() {
+        let calendarEvents = [
+            CalendarEvent(id: "1",
+                          calendarId: "1",
+                          description: "1",
+                          start: mockTime.now().addingTimeInterval(10 * .secondsInAMinute),
+                          stop: mockTime.now().addingTimeInterval(20 * .secondsInAMinute),
```

Comment:
I think this reads better
```suggestion
                          start: mockTime.now().addingTimeInterval(TimeInterval(minutes: 10)),
                          stop: mockTime.now().addingTimeInterval(TimeInterval(minutes: 20)),
```